ZuluContent/Items/Skill Items/Tinkering/AlchemicalSymbols.cs

```diff
@@ -0,0 +1,310 @@
+namespace Server.Items
+{
+    public class AlchemicalSymbol1 : Item
+    {
+        [Constructible]
+        public AlchemicalSymbol1() : base(0x181D)
```

Comment:
Really not a fan of having numbers in class names, does each symbol have an actual name we can use?
Is there a specific reason to create a particular symbol or can they be all one item that randomizes its ItemId on creation?

Reply:
I think the idea is that if you use them with Inscription each one will have a different effect, so randomizing will not be possible. Also there are no particular names, they're all just called "alchemical symbol" :/